include/quicly.h

```diff
@@ -431,6 +435,24 @@ struct st_quicly_default_scheduler_state_t {
     quicly_linklist_t blocked;
 };
 
+struct st_quicly_stateless_reset_t {
+    uint8_t *token;
+    uint8_t _buf[QUICLY_STATELESS_RESET_TOKEN_LEN];
+};
+
+/**
+ * records an extra CID given by the peer via NEW_CONNECTION_ID frame
+ */
+struct st_quicly_spare_cid_t {
+    /**
+     * indicates whether this record holds an active (given by peer and not retired) CID
+     */
+    int is_active;
+    uint64_t sequence;
+    quicly_cid_t cid;
+    struct st_quicly_stateless_reset_t stateless_reset;
```

Comment:
Let's use `uint8_t [QUICLY_STATELESS_RESET_TOKEN_LEN]` here. Please refer to my comment below for the rationale.

Reply:
👍 Done.